service/api_logic/news_logic.py

```diff
@@ -1,29 +1,36 @@
 import json
+from enum import Enum
+import pandas as pd
 from flask import Response
 from sqlalchemy import desc
 from database.models import News
 from database.azure_blob_storage.save_get_blob import blob_get_news
-from sqlalchemy.sql.expression import ClauseElement
-from exept.handle_exeptions import handle_exceptions
-from service.api_logic.scripts import get_sport_by_name
-from database.session import SessionLocal
 from logger.logger import Logger
+from service.api_logic.helpers.calculating_helper import CalculatingHelper
+
+
+class LimitsConsts(Enum):
```

Comment:
For the future: This should be in some separate place. All your general stuff should be separated

Don't fix it, let it be